.github/workflows/ci.yml

```diff
@@ -1,14 +1,23 @@
-name: Build
+name: Continuous Integration
```

Comment:
Based on the other repositories I think this should be just `test`.
https://github.com/surrealdb/surrealdb.js/blob/main/.github/workflows/test.yml#L1
https://github.com/surrealdb/surrealdb.go/blob/main/.github/workflows/test.yml#L1

Reply:
Renamed to `test`